ui/app/AppLayouts/Wallet/views/AssetsDetailView.qml

```diff
@@ -170,6 +177,15 @@ Item {
                     //},
                     //pan:{enabled:true,mode:'x'},
                     tooltips: {
+                        format: {
+                            enabled: graphDetail.selectedGraphType === AssetsDetailView.GraphType.Balance,
+                            callback: function (value) {
+                                return graphDetail.dateToShortLabel(value)
+                            },
+                            valueCallback: function(value) {
+                                return LocaleUtils.currencyAmountToLocaleString({ amount: value.toFixed(2), symbol: RootStore.currencyStore.currentCurrencySymbol })
```

Comment:
This (`toFixed(2)`) is probably why the number is formatted wrong... see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
You can pass `options {displayDecimals: 2}` to get the same result in the correct way